README.md

```diff
@@ -97,16 +97,11 @@ Daylight provides detailed specifications of the [SMILES](http://www.daylight.co
 
 # Frequently asked questions (FAQ)
 
-See [`Chemical-starting-points.md`](Chemical-starting-points.md) for answers to a variety of common problems, such as:
+See [`FAQ.md`](FAQ.md) for answers to a variety of common problems, such as:
 * Why do I need to provide molecules corresponding to the components of my system, or a `Topology` with bond orders?
 * Can I use an AMBER, CHARMM, or gromacs topology/coordinate file as a starting point for applying a SMIRNOFF force field?
 * What if I am starting from a PDB file?
 
 # Contributors
 
-* [David L. Mobley (UCI)](https://github.com/davidlmobley)
-* [John D. Chodera (MSKCC)](https://github.com/jchodera)
-* [Caitlin Bannan (UCI)](https://github.com/bannanc)
-* [Camila Zanette (UCI)](https://github.com/camizanette)
-* [Christopher I. Bayly (OpenEye)](https://github.com/cbayly13)
-* [Nathan M. Lim (UCI)](https://github.com/nathanmlim)
+For a full list of contributors, see the [GitHub Contributors page](https://github.com/openforcefield/openforcefield/graphs/contributors).
```

Comment:
I saw you put this in the news announcement as well. I think it's a good way to handle contributors.